joomla/joomla-cms
# Append csrf token not only to post requests

Pull Request for Issue # .

### Summary of Changes

Append the csrf token not only in post requests.


### Testing Instructions

Use joomla.request() with a Patch, Put or Delete request and see the csrf token appended to the request header
